### DISCUSSION TITLE
FNIRSI GC-01: HV tube profile setup

### DISCUSSION THREAD

**Discussion by faismoipeur** [Gissio/radpro#160] [Q&A] [answered]
Hello,

I Have an GC-01/ M4011 tube with an APM32(Geehy) chip. (2.0.3 and 2.0.2 tested!)
So, when i choose "energy-saving", 5 min later, it's crying that the tube is gone.

Have you tested interresting values for PMW freq and duty cycle ? 

Thank you in advance. 

Yann.

**Comment by dc1rdb** [comment]
Yann,
setting the right PWM parameters on the non-regulated GC-01 can be a bit tricky. I ended up with 5 kHz/2.75% on mine (also APM chip and M4011). That setting gives me enough reserve even for dose rates > 500uSv/h. Ideally, you should measure the voltage across the tube with a suitable HV probe to make sure to stay within spec for your tube.
Hope this helps.
Robert

**Reply by faismoipeur** [reply]
Je vais tester..

**Reply by faismoipeur** [reply]
Hi, 
10kHz/12% seems to work on mine. 5kHz/2.5%-90% doesn't work at all. I have tested this parameters only with my backgroung rate (~0.08µSv/h) against an BR-6. I have no radiation source for testing high dose rate.
Thanks.
Yann.

**Reply by faismoipeur** [reply, marked as answer]
~32h Autonomy with this parameters. With the original battery : https://ibb.co/jMTjtNY

**Reply by Gissio** [reply]
faismoipeur: it my understanding that you are a native French speaker. Could you check the French translation in 2.2beta3?

The strings are available here: https://github.com/Gissio/radpro/blob/main/platform.io/src/strings/fr.h

If you are willing to help us with these, please follow the discussion here: https://github.com/Gissio/radpro/discussions/241
